### PR TITLE
Remove committee check in

### LINKED ISSUE #284: Remove committee check in
The committee check in just pauses vault deposits until the committee starts them for the first time. This seems practically the same as having a 0 address check, which is quite unnecessary. Also, note that we could just start vaults with depositPaused as true until governance starts them to give some time to do checks before starting the vault. Also, not that currently, the committee check in is only for the first time the committee is set, but if after check in the committee is replaced, there is no need committee check in which seems inconsistent.

### PR DESCRIPTION
**Pull request by ben-kaufman**
Fix #284

### CODE REVIEW

File: contracts/vaults/Params.sol [hunk 1]
```diff
@@ -6,20 +6,10 @@ import "./Base.sol";
 contract Params is Base {
 
     /**
-    * @notice Set new committee address. Can be called by existing committee if it had checked in, or
-    * by the governance otherwise.
+    * @notice Set new committee address. Can be called only by the existing committee.
```
Comment (lirona) [inline contracts/vaults/Params.sol:9]:
```suggestion
    * @notice Called by the existing committee to set a new committee address.
```

Comment (lirona) [reply contracts/vaults/Params.sol:9]: just for consistency, i wrote everything that way

**Review by lirona — COMMENTED** [review]
lgtm, accept if i understood correctly, we said we also always initialize the vaults with deposits paused, and have committee also be able to change paused status

**Comment by jellegerbrandy** [comment]
I am closing this - in the end it seems better to keep the committee checkin to preserve non-rugpull safety :)

**Pull request closed without merging**